runtime/autoload/provider/ruby.vim

```diff
@@ -47,8 +43,24 @@ function! provider#ruby#Call(method, args) abort
   return call('rpcrequest', insert(insert(a:args, 'ruby_'.a:method), s:host))
 endfunction
 
+function! s:detect()
+  if exists("g:ruby_host_prog")
+    return g:ruby_host_prog
+  elseif has('win32')
```

Comment:
Use `endif` because of `return`.

Reply:
Without `:elseif` I'd even need more lines. :> I think that's just a style thing.